client/src/components/nav_comp/RightNavLogged.js

```diff
@@ -0,0 +1,104 @@
+//contains dropdown with log out for right side of nav bar
+
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import Button from '@material-ui/core/Button';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+import Grow from '@material-ui/core/Grow';
+import Paper from '@material-ui/core/Paper';
+import Popper from '@material-ui/core/Popper';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuList from '@material-ui/core/MenuList';
+import { Redirect } from 'react-router-dom';
+import { logout } from '../../store/auth';
+// import { Menu } from '@material-ui/core';
+// import { makeStyles } from '@material-ui/core/styles';
+
+// import { Link } from 'react-router-dom';
+    
+function RightNav(){
+    const dispatch = useDispatch();
+    const [open, setOpen] = React.useState(false);
+    const anchorRef = React.useRef(null);
+  
+    const handleToggle = () => {
+      setOpen((prevOpen) => !prevOpen);
+    };
+  
+    const handleClose = (event) => {
+      if (anchorRef.current && anchorRef.current.contains(event.target)) {
+        return;
+      }
+  
+      setOpen(false);
+    };
+
+    const handleLogOut = (event) => {
+      if (anchorRef.current && anchorRef.current.contains(event.target)) {
+          return
+    }
+    setOpen(false);
+    
+    dispatch(logout())
+    return <Redirect to="/login" />
```

Comment:
Being super knit-picky and just saw this spacing. 😅 

```suggestion
        }
        setOpen(false);
    
        dispatch(logout())
        return <Redirect to="/login" />
```